deepmd/dpmodel/output_def.py

```diff
@@ -228,6 +232,11 @@ def __init__(
     def size(self):
         return self.output_size
 
+    def squeeze(self, dim):
+        # squeeze the shape on given dimension
+        if -len(self.shape) <= dim < len(self.shape) != 1 and self.shape[dim] == 1:
```

Comment:
why `-len(self.shape)`

Reply:
because the dim may be negative, e.g. -1.